README.md

```diff
@@ -3,21 +3,75 @@
 ### High level constructs for Kubernetes
 
 ![Stability:Beta](https://img.shields.io/badge/stability-beta-orange)
-<!-- [![build](https://github.com/cdk8s-team/cdk8s-plus-17/workflows/release/badge.svg)](https://github.com/cdk8s-team/cdk8s-plus-17/actions/workflows/release.yml)
-[![npm version](https://badge.fury.io/js/cdk8s-plus-17.svg)](https://badge.fury.io/js/cdk8s-plus-17)
-[![PyPI version](https://badge.fury.io/py/cdk8s-plus-17.svg)](https://badge.fury.io/py/cdk8s-plus-17)
-[![NuGet version](https://badge.fury.io/nu/Org.Cdk8s.Plus17.svg)](https://badge.fury.io/nu/Org.Cdk8s.Plus17)
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.cdk8s/cdk8s-plus-17/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.cdk8s/cdk8s-plus-17) -->
+[![cdk8s-plus-22](https://img.shields.io/github/workflow/status/cdk8s-team/cdk8s-plus/release-k8s.22?label=cdk8s-plus-22&logo=GitHub)](https://github.com/cdk8s-team/cdk8s-plus/actions/workflows/release-k8s.22.yml)
+[![cdk8s-plus-21](https://img.shields.io/github/workflow/status/cdk8s-team/cdk8s-plus/release-k8s.21?label=cdk8s-plus-21&logo=GitHub)](https://github.com/cdk8s-team/cdk8s-plus/actions/workflows/release-k8s.21.yml)
+[![cdk8s-plus-20](https://img.shields.io/github/workflow/status/cdk8s-team/cdk8s-plus/release-k8s.20?label=cdk8s-plus-20&logo=GitHub)](https://github.com/cdk8s-team/cdk8s-plus/actions/workflows/release-k8s.20.yml)
 
-**cdk8s+** is a software development framework that provides high level abstractions for authoring Kubernetes applications.
-Built on top of the auto generated building blocks provided by [cdk8s](../cdk8s), this library includes a hand crafted *construct*
-for each native kubernetes object, exposing richer API's with reduced complexity.
+| k8s version | npm (JS/TS) | PyPI (Python) | NuGet (C#) | Maven (Java) | Go |
+| --- | --- | --- | --- | --- | --- |
+| 1.20.0 | [Link](https://www.npmjs.com/package/cdk8s-plus-20) | [Link](https://pypi.org/project/cdk8s-plus-20/) | [Link](https://www.nuget.org/packages/Org.Cdk8s.Plus20) | [Link](https://search.maven.org/artifact/org.cdk8s/cdk8s-plus-20) | [Link](https://pkg.go.dev/github.com/cdk8s-team/cdk8s-plus-go/cdk8splus20) |
```

Comment:
Maybe make the Go links point to the relevant GitHub tag? This way all links point to the published artifact.

Reply:
Done.